h5py_wrapper/wrapper.py

```diff
@@ -259,6 +259,8 @@ def _load_dataset(f, lazy=False):
     else:
         try:
             value_type = f.attrs['_value_type']
+            if isinstance(value_type, bytes):
+                value_type = str(value_type, 'utf-8')
```

Comment:
why do we need to provide `'utf-8` here, shouldn't `str(value_type)` be enought? also `str('test', 'utf-8')` throws an error when I try it under python 3.